charts/pravega/values.yaml

```diff
@@ -2,7 +2,7 @@
 # This is a YAML-formatted file.
 # Declare variables to be passed into your templates.
 
-version: 0.5.0
+version: latest
```

Comment:
"latest" version is not accepted by webhook in it validation. version can be changed to `0.7.0`